bins/stackabled/src/handlers/stacklets.rs

```diff
@@ -0,0 +1,41 @@
+use axum::{extract::Path, routing::get, Json, Router};
+use stackable::kube::k8s_openapi::{
+    apimachinery::pkg::apis::meta::v1::ObjectMeta, serde::Serialize,
```

Comment:
Do we want to really want to use the re-exported `k8s_openapi` and `serde` crates. I favor direct imports and dependencies in the `Cargo.toml` file.

Reply:
Happy to add a direct dependency on `serde`. `k8s-openapi` is a bit weird since we need to have one canonical place that decides which version features to use.

Reply:
Then again, this is more of a temp solution anyway, `Stacklet` should be moved to libcockpit at some point.

Reply:
ba8c284cd337680db604d546438bd3db84ebbeec

Reply:
> Happy to add a direct dependency on `serde`. `k8s-openapi` is a bit weird since we need to have one canonical place that decides which version features to use.

We can "pin" the version feature used across all workspace members using a workspace dependency. With that, all members using that dependency, use the same version.

Reply:
True, good catch.

Reply:
b00fbbe350fd99562fb6f64c09166271934d7076

Reply:
Neat! Resolved.